_includes/header.html

```diff
@@ -2,14 +2,9 @@
   <div class="container-fluid">
     <nav class="row middle-xs between-xs">
       <a class="mainlogo" href="{{site.baseurl}}/" >OpenTechSchool</a>
-      <div class="col-xs-12 col-sm-6 row end-xs">
-        <ul class="social-links row col-xs-5 between-xs">
-          {% include _render_nav.html nav=site.data.nav.social %}
-        </ul>
-
-        <ul class="main row col-xs-9 between-xs">
-          {% include _render_nav.html nav=site.data.nav.main %}
-        </ul>
+      <div class="main row between-xs col-xs-6">
```

Comment:
Did we check how this look like in mobile? Does it need some adjustment or is good?

Reply:
Hmmm.. It looks... under construction.. With the 'toggle device' tool you can only have a little notion of the final effect I think... There is no way we can run the test page on a real smartphone? Can I generate the link some how so I could open in another device than the computer rendering the site through Jekyll? I doubt it...
I'll play with it a bit more and see if it should be changed..